Source/Core/PolylinePipeline.js

```diff
@@ -286,15 +286,29 @@ define([
         }
         //>>includeEnd('debug');
 
+        var length = positions.length;
         var ellipsoid = defaultValue(options.ellipsoid, Ellipsoid.WGS84);
         var height = defaultValue(options.height, 0);
+
+        if (length < 1) {
+            return [];
+        } else if (length === 1) {
+            var p = ellipsoid.scaleToGeodeticSurface(positions[0], scaleFirst);
```

Comment:
Neither is `cartesian` below. Can you check the rest of the scratch variables to see that they are defined at the file scope?